node/formats/eszip.ts

```diff
@@ -52,6 +53,11 @@ const bundleESZIP = async ({
   try {
     await deno.run(['run', ...flags, bundler, JSON.stringify(payload)], { pipeOutput: true })
   } catch (error: unknown) {
+    const npmImportError = checkNpmImportError(error)
+    if (npmImportError) {
+      throw npmImportError
```

Comment:
`wrapBundleError` is responsible for setting the `customErrorInfo` property, which is then used to distinguish user errors from system errors. By not wrapping `npmImportError`, we'll start incorrectly treating these as system errors.

Reply:
that's good to know! fixed in https://github.com/netlify/edge-bundler/pull/195/commits/d49211a50796209f346379a583ae972070c32230 and added a comment about that in https://github.com/netlify/edge-bundler/pull/195/commits/fd38b1f6f7e9a74cc983694312184ccc64a8bda8